packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -254,26 +254,22 @@ Future<XcodeBuildResult> buildXcodeProject({
   // copied over to a location that is suitable for Xcodebuild to find them.
   final Directory appDirectory = fs.directory(app.appDirectory);
   await _addServicesToBundle(appDirectory);
-  final InjectPluginsResult injectPluginsResult = injectPlugins();
-  final bool hasFlutterPlugins = injectPluginsResult.hasPlugin;
   final String previousGeneratedXcconfig = readGeneratedXcconfig(app.appDirectory);
 
   updateXcodeGeneratedProperties(
     projectPath: fs.currentDirectory.path,
     buildInfo: buildInfo,
     target: target,
-    hasPlugins: hasFlutterPlugins,
```

Comment:
Depending on if I understood the PR description intent correctly, it sounds like you want to move this call to `updateXcodeGeneratedProperties` to `flutter packages get` in a future PR too?

Reply:
I think you are right. I left that out because 1) it was not necessary on a newly created project, and 2) the properties include information like the build mode which is not available when we run `flutter packages get`.

But 1) is because `flutter create` generates the `Generated.xcconfig` file with some default settings along with a .gitignore entry that causes it not to be present on subsequent git checkout. And for 2) those default settings can be used during `flutter packages get` too.